.gitignore

```
@@ -48,6 +48,9 @@ ch05/07_gpt_to_llama/Llama-3.2-1B
 ch05/07_gpt_to_llama/Llama-3.2-1B-Instruct
 ch05/07_gpt_to_llama/Llama-3.2-3B
 ch05/07_gpt_to_llama/Llama-3.2-3B-Instruct
+ch05/10_llm-training-speed/middlemarch.txt
```

Thanks for the PR! What do you think about the following change to make it a bit more general?

```suggestion
ch05/10_llm-training-speed/*.txt
```

---

Your choice - both solutions would work. It's the only file that is being downloaded:
https://github.com/rasbt/LLMs-from-scratch/blob/b7893457da9fe66ebe04aebbb9797e598f7f0e91/ch05/10_llm-training-speed/00_orig.py#L424

---

I totally forgot that this was the dataset I used here, I thought, for some reason, it's a dataset you locally used lol. All good then!